### PR TITLE
fix: set max-width for Toast message

### LINKED ISSUE #9049: [Bug]: Toast should be having a max width
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Toast messages are too lengthy

image (1)

### Steps To Reproduce

Use `showAlert` to show a large message

### Environment

Release

### Version

Self-Hosted

### PR DESCRIPTION
**Pull request by aswathkk**
## Description

Set max-width for Toast.

Fixes #9049 

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

Use `showAlert` to create a lengthy message

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

## Test coverage results :test_tube:
<details><summary>:red_circle: Total coverage has decreased</summary>

    // Code coverage diff between base branch:release and head branch: fix/toast-lengthy-content 
Status | File | % Stmts | % Branch | % Funcs | % Lines 
 -----|-----|---------|----------|---------|------ 
 :red_circle: | total | 54.77 **(0)** | 36.62 **(-0.01)** | 33.48 **(0.01)** | 55.3 **(-0.01)**
 :green_circle: | app/client/src/components/ads/Toast.tsx | 77.78 **(0.42)** | 63.64 **(1.14)** | 80.95 **(0.95)** | 77.78 **(0.42)**
 :red_circle: | app/client/src/selectors/commentsSelectors.ts | 83.61 **(-1.64)** | 61.76 **(-2.95)** | 73.33 **(0)** | 88.24 **(-2.35)**
 :red_circle: | app/client/src/utils/autocomplete/TernServer.ts | 52.04 **(-0.22)** | 40.42 **(-0.83)** | 34.48 **(0)** | 55.96 **(-0.26)**</details>

### CODE REVIEW

**Review by rishabhsaxena — DISMISSED** [review]
LGTM, although wondering if we should keep the max-width a bit on the lower side? Since there isn't a way to add line breaks to the message currently

File: app/client/src/components/ads/Toast.tsx [hunk 1]
```diff
@@ -60,6 +60,7 @@ const ToastBody = styled.div<{
   width?: string;
 }>`
   width: ${(props) => props.width || "fit-content"};
+  max-width: 500px;
```
Comment (somangshu) [inline app/client/src/components/ads/Toast.tsx:63]: I agree to @rishabhsaxena, I remember that the width before was fixed at `260px`, we should use a similar value for the max-width

Comment (somangshu) [reply app/client/src/components/ads/Toast.tsx:63]: On deploy preview this look alright, It's not so big. @rishabhsaxena do you this looks alright
Screenshot 2021-11-09 at 4 17 30 PM

**Comment by somangshu** [comment]
https://github.com/appsmithorg/appsmith/actions/runs/1439092299

File: app/client/src/components/ads/Toast.tsx (hunk 1 above)
Comment (aswathkk) [reply app/client/src/components/ads/Toast.tsx:63]: @somangshu In that case, the action button for undo can come in the next line

**Comment by aswathkk** [comment]
@somangshu @rishabhsaxena 
A widget's name can be a maximum of 30 characters. Here's how the message look like if we do undo/redo for a widget that has 30 character length name. It's around 440px length. If we reduce the width, the action buttons can go below the text.
Screenshot 2021-11-09 at 4 21 59 PM

**Comment by rishabhsaxena** [comment]
An alternate solution could be adding a `max-width` prop to the `undo / redo` toasts as a quick fix, and keeping the default as before?

**Comment by somangshu** [comment]
> @aswathkk an alternate solution could be adding a `max-width` prop to the `undo / redo` toasts as a quick fix, and keeping the default as before?

If we are able to keep the styles separate for both the conditions aforementioned, We should go about doing it

**Comment by aswathkk** [comment]
PR #8955 caused this issue. Before that, the width was 264px. I set it as the default value for max-width now.

**Review by rishabhsaxena — APPROVED** [review]

**Review by somangshu — APPROVED** [review]

**Comment by somangshu** [comment]
https://github.com/appsmithorg/appsmith/actions/runs/1439803060

**Pull request merged**